mongodb_store/scripts/config_manager.py

```diff
@@ -172,12 +172,14 @@ def flatten(d, c="", f_name="" ):
                     rospy.set_param(name,val)
                 else:
                     rospy.logerr("Unable to set parameter %, its value is None.", name)
+
+        warn_if_stored_param_has_no_default = rospy.get_param("~warn_if_stored_param_has_no_default")
```

Comment:
It might be valuable to provide a default here. Otherwise you are forcing the user to provide a parameter when starting this without using roslaunch.